src/topology/continuous_function/algebra.lean

```diff
@@ -340,6 +363,14 @@ instance module : module R C(α, M) :=
 
 variables (R)
 
+/-- Composition on the left by a continuous linear map, as a `linear_map`.
+Similar to `linear_map.comp_left`. -/
+@[simps] protected def _root_.continuous_linear_map.comp_left_continuous (α : Type*)
+  [topological_space α] (g : M →L[R] M₂) :
+  C(α, M) →ₗ[R] C(α, M₂) :=
+{ map_smul' := λ c f, ext $ λ x, g.map_smul' c _,
+  .. g.to_linear_map.to_add_monoid_hom.comp_left_continuous α g.continuous }
```

Comment:
Am I  missing something here? Can this also be built as follows?
```suggestion
g.to_linear_map.comp_left_continuous α g.continuous
```

Reply:
Did you try it? :)
```
type mismatch, term
  add_monoid_hom.comp_left_continuous α g.to_linear_map.to_add_monoid_hom _
has type
  C(α, M) →+ C(α, M₂)
but is expected to have type
  C(α, M) →ₗ[R] C(α, M₂)
```

Reply:
Is there a reason you didn't define `linear_map.comp_left_continuous`?

Reply:
Because mathlib has the definition `continuous_linear_map`, but not `continuous_add_hom`, `continuous_ring_hom`, or `continuous_alg_hom`.